Extension/src/main.ts

```diff
@@ -73,63 +47,31 @@ export async function activate(context: vscode.ExtensionContext): Promise<CppToo
     // Initialize the DebuggerExtension and register the related commands and providers.
     DebuggerExtension.initialize(context);
 
-    await processRuntimeDependencies();
+    const info: PlatformInformation = await PlatformInformation.GetPlatformInformation();
+    await makeBinariesExecutable();
+    sendTelemetry(info);
```

Comment:
I was thinking we only sent this after downloading binaries (i.e. indicating an installation). We may want to rethink what we do with this telemetry, or rename the event to more accurately depict the event.